src/js/hca/components/military-service/ServiceInformationSection.jsx

```diff
@@ -58,29 +58,29 @@ class ServiceInformationSection extends React.Component {
               value={this.props.data.lastServiceBranch}
               onValueChange={(update) => {this.props.onStateChange('lastServiceBranch', update);}}/>
 
-          <DateInput required
-              errorMessage="Entry date must be greater than 15 years of age"
-              validation={isValidEntryDateField(this.props.data.lastEntryDate, this.props.data.veteranDateOfBirth)}
-              label="Last entry date"
+          <ErrorableDate required
```

Comment:
Same question as above, should all these be `ErrorableCurrentOrPastDate`?

I think DateInput by default only allows past dates (which is the reason we have several date components, we were trying to refactor and make things more obvious).